mvpa2/generators/permutation.py

```diff
@@ -231,6 +231,9 @@ def _permute_chunks_sanity_check(in_pattrs, chunks, uniques):
 
     def _permute_chunks(self, limit_idx, in_pattrs, out_pattrs, chunks=None):
         # limit_idx is doing nothing
+        
+        if chunks == None:
+            raise RuntimeError("Missing 'chunk_attr' for strategy='chunk'")
```

Comment:
comparison to none  better uses "is",  not ==
 and ValueError  would be  more  appropriate here  as well

 otherwise -  thanks for the PR  with the test!